modules/ROOT/pages/fd-connection-view-audit-logs.adoc

```diff
@@ -0,0 +1,24 @@
+= View Audit Logs for Connections That are Shared
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+Although only the owner of a shared connection can edit or delete it, someone might illicitly acquire the credentials and edit or delete one or more shared connections. Someone also might maliciously use another user's credentials to create a shared connection and siphon data to an unapproved system.
+
+To help you identify such scenarios, Flow Designer creates an audit-log message for every creation, change to, and deletion of a shared connection within your business group. Each message includes the following information:
+
+* The time and date of the action
+* The product in which the action was taken (i.e., Flow Designer)
+* The type of change, which is always "Connection"
+* The type of action: Create, Edit, or Delete,
+* The type of connector that the connection was for, such as the Database connector.
+* The user ID that was used to take the action
```

Comment:
Maybe, I'll add that you can download the payload in the last column, that contains more detailed information about the connection... not credentials, but the ID of the connection, the user that created the connection, creation and modification date, type of connector, among others

Reply:
Added.